server.js

```diff
@@ -2,3 +2,15 @@
 const { ScrollHub } = require("./ScrollHub.js")
 
 new ScrollHub().startAll()
+
+if ('serviceWorker' in navigator) {
```

Comment:
Could you add some comments explaining what this is doing? What is a service-worker? Where can someone learn more?